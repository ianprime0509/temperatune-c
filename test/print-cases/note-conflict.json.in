{
  "name": "Note has conflicting definitions",
  "octaveBaseName": "C",
  "referencePitch": 440,
  "referenceName": "A",
  "referenceOctave": 4,
  "notes": {
    "A": ["E", 5],
    "C": ["A", 500],
    "E": ["A", 5]
  }
}
