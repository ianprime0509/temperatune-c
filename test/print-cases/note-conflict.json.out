temperatune: found conflicting offset for 'E'
